oauthproxy.go

```diff
@@ -551,6 +552,9 @@ func (p *OAuthProxy) serveHTTP(rw http.ResponseWriter, req *http.Request) {
 		p.provider.Data().RedeemURL.Path)
 	req = req.Clone(ctx)
 
+	span, ctx := trace.ContextBoundTrace(ctx, "ListProjects")
```

Comment:
Change "ListProjects" to something more meaningful. We can use `req.URL.Path` for now.

Reply:
Done.